src/schema/position.rs

```diff
@@ -147,6 +266,7 @@ impl SchemaDefinitionPosition {
     fn insert_references(
         &self,
         schema_definition: &Node<SchemaDefinition>,
+        schema: &Schema,
```

Comment:
since schema contains schema definition do we need both?

Reply:
The second argument is meant to be the element being inserted for all the `insert_references()`-like functions in this file. Sometimes we also need `schema` too, in which case it's also an argument (mostly its used to check the type kind of a type, but in this PR we're also using it for meta-fields via the `apollo-rs` code).

When we have `schema`, we could just omit the 2nd argument as you've said, but we always have the element we're inserting at the callsites, so it saves us some map lookups if we just pass it along as an argument instead of re-fetching.